en/services/high_latency.md

```diff
@@ -1,27 +1,29 @@
 # High Latency Protocol
 
-High latency links (e.g. Iridium Satellite links) provide global connectivity, albeit with significant message latency and high cost-per-message.
-Generally the cost and latency means that high-latency links are only used when there is no lower-latency alternative, and when active the links should only send essential information.
+High latency links (e.g. Iridium Satellite links) provide global connectivity, albeit with significant message latency (> 1 second) and high cost-per-message.
+Generally the cost and latency means that high-latency links are only used when there is no lower-latency alternative, and when active the links should only send essential information or commands.
 
-The protocol provides a heartbeat-like message ([HIGH_LATENCY2](#HIGH_LATENCY2)) for transmitting just the most important telemetry at low rate, and a command ([MAV_CMD_CONTROL_HIGH_LATENCY](#MAV_CMD_CONTROL_HIGH_LATENCY)) for enabling/disabling the high latency link when needed (i.e. when no lower-latency link is available).
+The protocol provides a heartbeat-like message ([HIGH_LATENCY2](#HIGH_LATENCY2)) for transmitting just the most important telemetry at low rate, and a command ([MAV_CMD_CONTROL_HIGH_LATENCY](#MAV_CMD_CONTROL_HIGH_LATENCY)) for enabling/disabling the high latency link when needed (i.e. when no lower-latency link is available). Alternatively, the flight controller can be set to enable the high latency link by default (i.e. all links are active throughout the flight for redundancy purposes).
```

Comment:
Using the HL for redundancy sounds like cost for no benefit. If the LL is working the message will always be chucked. If it is not working, then the GCS should enable the HL. Where is this useful?

If it is OK then we can suggest it as OK, but not in the main paragraph describing the expected behaviour.

Reply:
Removed